rotkehlchen/chain/manager.py

```diff
@@ -1527,3 +1527,18 @@ def get_loopring_balances(self) -> Dict[Asset, Balance]:
                 aggregated_balances[asset] += balance
 
         return dict(aggregated_balances)
+
+    def get_chain_manager(
+        self,
+        blockchain: SupportedBlockchain,
+    ) -> Any:
+        if blockchain == SupportedBlockchain.ETHEREUM:
+            return self.ethereum
+        if blockchain == SupportedBlockchain.KUSAMA:
+            return self.kusama
+        if blockchain == SupportedBlockchain.POLKADOT:
+            return self.polkadot
+        if blockchain == SupportedBlockchain.AVALANCHE:
+            return self.avalanche
+
+        raise ValueError(blockchain.value)
```

Comment:
Don't raise `ValueError`. We don't use it in the codebase. Just leave it, due to typechecking this should never happen. If anything just put an `assert` here that this should never happen which will be optimized out at the binary creation

Reply:
:+1: done